include/nbl/ext/LumaMeter/CLumaMeter.h

```diff
@@ -90,7 +91,18 @@ class CLumaMeter : public core::TotalInterface
 		static core::SRange<const asset::SPushConstantRange> getDefaultPushConstantRanges();
 
 		//
-		static core::SRange<const video::IGPUDescriptorSetLayout::SBinding> getDefaultBindings(video::IVideoDriver* driver);
+		static core::SRange<const video::IGPUDescriptorSetLayout::SBinding> getDefaultBindings(video::ILogicalDevice* device);
+
+		//
+		static inline core::smart_refctd_ptr<video::IGPUPipelineLayout> getDefaultPipelineLayout(video::ILogicalDevice* device)
+		{
+			auto pcRange = getDefaultPushConstantRanges();
+			auto bindings = getDefaultBindings(device);
+			return device->createGPUPipelineLayout(
+				pcRange.begin(), pcRange.end(),
+				device->createGPUDescriptorSetLayout(bindings.begin(), bindings.end()), nullptr, nullptr, nullptr
+			);
+		}
```

Comment:
create the default pipeline in the ctor, here return only the smartpointer to it